ui/components/app/snaps/snap-ui-renderer/index.scss

```diff
@@ -84,4 +84,12 @@
       max-width: $width-screen-lg-min;
     }
   }
+
+  &__form {
+    .snap-ui-renderer__panel {
+      .mm-box--display-flex.mm-box--flex-direction-column {
```

Comment:
Wondering if this selector is too fragile, not all fields may use this specific class name in the future. Perhaps we could add a new class name to the inputs if they are contained within a field?